Sources/Commands/SwiftTool.swift

```diff
@@ -433,17 +433,8 @@ public class SwiftTool {
         self.sharedConfigurationDirectory = try getSharedConfigurationDirectory(options: self.options, observabilityScope: self.observabilityScope)
         self.sharedCacheDirectory = try getSharedCacheDirectory(options: self.options, observabilityScope: self.observabilityScope)
 
-        // set verbosity globals.
-        // TODO: get rid of this global settings in TSC
-        switch self.logLevel {
-        case .debug:
-            TSCUtility.verbosity = .debug
-        case .info:
-            TSCUtility.verbosity = .verbose
-        case .warning, .error:
-            TSCUtility.verbosity = .concise
-        }
-        Process.verbose = TSCUtility.verbosity != .concise
+        // set global process logging handler
+        Process.loggingHandler = { self.observabilityScope.emit(debug: $0) }
```

Comment:
this means the process arguments will only be logged in _debug_ mode or iow require --vv to be shown

Reply:
I think this is good — it's been annoying that _all_ process invocations such as `vtool` to discover SDK info or manifest compilation gets logged with `-v`.

Reply:
Probably something to note in the release notes.